ingestion_tools/scripts/data_validation/tests/test_frame.py

```diff
@@ -52,17 +52,54 @@ def test_extensions(self, frames_files: List[str]):
     def test_consistent(self, frames_headers: Dict[str, Union[List[tifffile.TiffPage], MrcInterpreter]]):
         return helper_tiff_mrc_consistent(frames_headers)
 
-    ### END Self-consistency tests ###
+    ### BEGIN MDOC tests ###
+    @allure.title("Mdoc: tilt angles are within the expected range [-90, 90].")
+    def test_frames_mdoc_range(self, frames_mdoc: pd.DataFrame):
+        assert frames_mdoc["TiltAngle"].min() >= -90
+        assert frames_mdoc["TiltAngle"].max() <= 90
+
+    @allure.title("Mdoc: number of mdoc tilt angles equals number of frames files.")
+    def test_mdoc_frames(self, frames_mdoc: pd.DataFrame, frames_files: List[str]):
```

Comment:
I believe this is an existing test, but is it a valid case, when there are lesser number of entries in the mdoc than the actual number of frames?